nimbus.fml.yaml

```diff
@@ -342,6 +348,15 @@ features:
             URGENT:
               priority: 100
               max-display-count: 10
+          messages:
+            default-browser:
+              title: Default Browser/DefaultBrowserCard.Title
+              text: Default Browser/DefaultBrowserCard.Description
+              action: "MAKE_DEFAULT_BROWSER"
+              trigger: ["I_AM_NOT_DEFAULT_BROWSER", "SUPPORTS_DEFAULT_BROWSER"]
+              style: "FALLBACK"
+              # Use string interpolation because $TABLE_NAME/$KEY doesn't exist in en.lproj/Default Browser.strings
+              button-label: \(String.FirefoxHomepage.HomeTabBanner.EvergreenMessage.HomeTabBannerButton)
```

Comment:
This is horrible :) , but I understand why it's here, and am amazed that it works. Well done :)

Please could you file an issue against the localization subsystem to ensure [`Bundle.localizedString`][1] works for all locales.

[1]: https://developer.apple.com/documentation/foundation/bundle/1417694-localizedstring

Reply:
Yes I will open an issue to fix this